src/components/order-summary/order-summary.js

```diff
@@ -0,0 +1,30 @@
+import orderSummaryTemplate from './order-summary.hbs';
+import orderSummaryItemsTemplate from './order-summary-items.hbs';
+import { getCartFromStorage } from '../cart/cart';
+
+let orderSummaryItems;
+
+const globalVariables = {
+  rootPath: (
+    window.location.pathname === '/index.html'
+    || window.location.pathname === '/accessible-ecommerce-demo/'
+    || window.location.pathname === '/accessible-ecommerce-demo/index.html'
+  ) ? '.' : '..'
+};
```

Comment:
Commenting here for posterity: Would be good to come up with a better way of managing this rootPath thing in a way that's actually global and doesn't require a new definition for every page. We don't have to fix it here but it might be a good candidate for a future 'kitchen sink hackathon'